submissions/reactjs/assignment2.MD

```diff
@@ -3,7 +3,7 @@
 | Shubham Karvariya              | 2203031080004        |                   |             |
 | Ayush                          | 2203031080028        |                   |             |
 | Mansha Soni                    | 2203031080121        |                   |             |
-| Bhavin Parmar                  | 2203031080042        |                   |             |
+| Bhavin Parmar                  | 2203031080042        | [Assignmnet 32](https://github.com/bhavinbvn/React/tree/main/src)                  | [Github](https://github.com/bhavinbvn/React/)            |
```

Comment:
Please follow the repository structure mentioned in the problem statement document